tests/terraform/runner/resources/plan_and_tf_combine_graph_with_missing_resources/tfplan.json

```diff
@@ -264,7 +263,6 @@
                "grant":true,
                "hosted_zone_id":true,
                "id":true,
-               "lifecycle_rule":true,
```

Comment:
Shouldn't this be now `true_after_unknown`?

Reply:
No, this is the plan file itself. I erased it because now the check that checks existence of lifecycle rule passes so we I needed to align the test with this case.